sentences/de/climate_HassClimateGetTemperature.yaml

```diff
@@ -7,4 +7,4 @@ intents:
           - "Wie hoch ist [<area>] die Temperatur"
           - "Wie hoch ist die Temperatur (von | vom) <name>"
           - "Wie hoch ist die Temperatur vom Thermostat <area>"
-          - "Wie warm ist es <area>"
\ No newline at end of file
+          - "Wie warm ist es [<area>]"
```

Comment:
I thought no the area expansion should not be optional, since just "Wir warm ist es" (_How warm is it?_) seems incomplete without knowing the target area/room

Reply:
Just thought of the possibility where the device knows in which room you are. I reverted that for now.

Reply:
This possibility/feature is not there, yet... So we should not count on it 😉